docs/kubernetes/KubernetesIntegration.md

```diff
@@ -0,0 +1,38 @@
+# Integrating the Calico-Kubernetes Plugin
+
+This guide will take you through all of the configuration hooks needed to integrate the Calico-Kubernetes plugin into your cloud platform.
+
+## Setting Up Your Environment 
+   The Calico-Kubernetes plugin looks for three environment variables. If one is not set, it will assume a default value. For custom networks, you must configure these variables in the _kubelet_ environment, not the _shell_ environment. 
```

Comment:
Replace "For custom networks..." with "If you need to override the defaults, you must set these variables in the environment of the kubelet process."